pkg/nsx/services/vpc/vpc.go

```diff
@@ -185,13 +184,29 @@ func InitializeVPC(service common.Service) (*VPCService, error) {
 	return VPCService, nil
 }
 
+func (s *VPCService) GetVPCsByFreshNamespace(ctx context.Context, namespace string) []*model.Vpc {
```

Comment:
`GetVPCsByFreshNamespace` is not a good name, I understand you want to say this func will return existing VPCS for a given namesapce name, 
It's better to rename it as GetExistingVPCsByNamespace or GetCurrentVPCsByNamespace

`GetVPCsByNamespace` is to return all VPs in the store for a given namesapce name, including existing ones and deleted/staled VPCs